lib/utils.js

```diff
@@ -356,6 +399,20 @@ const utils = {
     return statusByFile;
   },
 
+  getShallowComparativeStatus: (localInfo, remoteInfo) => {
```

Comment:
This could be simplified to the previous comment of if the file exists locally and remotely.

Reply:
Since I'm no longer setting any 'status' key for local files, I'm going to remove this function and do the conflict check in getExporter.